application/hart0/hss_init.c

```diff
@@ -190,7 +190,7 @@ void HSS_PrintToolVersions(void)
 
 bool HSS_E51_Banner(void)
 {
-    mHSS_FANCY_PRINTF(LOG_STATUS,
+    mHSS_FANCY_PRINTF(LOG_STATUS, CRLF CRLF CRLF CRLF CRLF CRLF
```

Comment:
This is my test code. It is not necessary to make this change

Reply:
changed and verified build